core/deviceadaptor.cpp

```diff
@@ -120,10 +120,12 @@ RingBufferBase* DeviceAdaptor::findBuffer(const QString& name) const
 bool DeviceAdaptor::setStandbyOverride(bool override)
 {
     standbyOverride_ = override;
-    if (override) {
-        resume();
-    } else {
-        standby();
+    if (screenBlanked_) {
+        if (override) {
+            resume();
+        } else {
+            standby();
+        }
```

Comment:
While this probably works, I keep wondering how come these need to be adjusted all the time.

AFAIK there are three states (sensor started, standby override, display blanked) that affect whether the sensor should be powered on or off. Now all state changes seem to have custom evaluators. How about just updating the cached member data and then calling common evaluator with logic something like:

    if(started && (!blanked || override))
        resume();
    else
        standby();